<div style="text-align: center;"> <h3>Statistical Theory</h3>
<h5>Formative Assessment 2</h5>
<h5><u>By Romand Lansangan</u></h5>
    </div>
    
---

#### (3.49) Prove that:
$$
\sum_{j=1}^N (X_j - 1)^2 = \sum_{j=1}^N X_j^2 - 2\sum_{j=1}^N X_j + N
$$

Proof:

<b>Expanding the summand</b> we'll have,

$$
\sum_{j=1}^N (X_j - 1)^2 = \sum_{j=1}^N X_j^2 - 2\sum_{j=1}^N X_j + \sum_{j=1}^N 1
$$

Note that in $ \sum_{j=1}^N 1 $ we just ought to add 1 Nth times. Therefore,

$$
\sum_{j=1}^N (X_j - 1)^2 = \sum_{j=1}^N X_j^2 - 2\sum_{j=1}^N X_j + \sum_{j=1}^N N
$$

As desired $\square$

---
#### (3.51)
Two variables, U and V, assume the values $U_t = 3 , U_2 = -2 , U_3 = 5, and V_1 = -4, V_2 = -1 , V_3 = 6$ , respectively. Calculate

##### (a)
$$
\sum UV
$$

We could use python to calculate (a)

In [4]:
u_n = (3,-2,5)
v_n = (-4, -1, 6)

Let us multiply the each pair for (UV)

In [6]:
series_a = [u_n[i] * v_n[i] for i in range(len(u_n))]
series_a

[-12, 2, 30]

Getting the sum of the results,

In [8]:
sum(series_a)

20

Therefore,

$$
\begin{align*}
\sum UV &= (3 \times -4) + (-2 \times -1) + (5 \times 6) \\
        &= -12 + 2 + 30 \\
        &= 20
\end{align*}
$$

##### (b)
$$
\sum (U+3)(V-4)
$$

Similarly, we could also use python to compute this

In [11]:
u_n_plus_three = [u + 3 for u in u_n]
u_n_plus_three

[6, 1, 8]

In [12]:
v_n_minus_four = [v - 4 for v in v_n]
v_n_minus_four

[-8, -5, 2]

Then multiplying the two series or vector,

In [14]:
series_b = [u_n_plus_three[i] * v_n_minus_four[i] for i in range(len(u_n_plus_three))]
series_b

[-48, -5, 16]

In [15]:
sum(series_b)

-37

Therefore,


$$
\begin{align*}
\sum (U+3)(V-4) &= (3 + 3)(-4 -4) + (-2 + 3)(-1 - 4) + (5 + 3)(6 - 4) \\
        &= (6)(-8) + (1)(-5) + (8)(2) \\
        &= -48 - 5 + 16 \\
        &= -37
\end{align*}
$$

##### (c)
$$
\sum V^2
$$

In [18]:
series_c = [v**2 for v in v_n]
series_c

[16, 1, 36]

In [19]:
sum(series_c)

53

$$
\begin{align*}
\sum V^2 &= (-4)^2 + (-1)^2 + 6^2 \\
        &= 16 + 1 + 36 \\
        &= 53
\end{align*}
$$

##### (d)

$$
(\sum U ) (\sum V )^2
$$

In [22]:
sum_u_n = sum(u_n)
sum_u_n

6

In [23]:
sum_v_n = sum(v_n)
sum_v_n

1

In [24]:
sum_u_n * (sum_v_n**2)

6

$$
\begin{align*}
(\sum U ) (\sum V )^2 &= 6 \times 1^2 \\
        &= 6
\end{align*}
$$

##### (e)

$$
\sum UV^2
$$

In [27]:
series_e = [u_n[index] * series_c[index] for index in range(len(u_n))]
series_e

[48, -2, 180]

In [28]:
sum(series_e)

226

$$
\begin{align*}
\sum UV^2 &= (3)(-4)^2 + (-2)(-1)^2 + (5)6^2 \\
        &= 48 - 2 + 180 \\
        &= 226
\end{align*}
$$

##### (f)

$$
\sum (U^2 - 2V^2 + 2)
$$

In [31]:
u_n_squared = [u**2 for u in u_n]
u_n_squared

[9, 4, 25]

In [32]:
two_v_n_squared = [2*(v**2) for v in v_n]
two_v_n_squared

[32, 2, 72]

In [33]:
series_f = [u_n_squared[index] - two_v_n_squared[index] + 2 for index in range(len(u_n_squared))]
series_f

[-21, 4, -45]

In [34]:
sum(series_f)

-62

$$
\begin{align*}
\sum (U^2 - 2V^2 + 2) &= (3^2 - 2(-4)^2 + 2) + ((-2)^2 - 2(-1)^2 + 2) + (5^2 - 2(6)^2 + 2) \\
        &= -21 + 4 -45\\
        &= -62
\end{align*}
$$

##### (g)
$$
\sum (U/V)
$$

In [37]:
series_g = [u_n[index] / v_n[index] for index in range(len(u_n))]
series_g

[-0.75, 2.0, 0.8333333333333334]

In [38]:
sum(series_g)

2.0833333333333335

$$
\begin{align*}
\sum (U/V) &= (3/-4) + (-2/-1) + (5/(6) \\
        &= -0.75 + 2.0 + 0.8\overline{33}\\
        &\approx 2.08\overline{33}
\end{align*}
$$

---
### (3.90)

Find the geometric mean of the sets <b>(a) 3, 5, 8, 3, 7, 2 </b> and <b>(b) 28.5, 73.6, 47.2, 31.5, 64.8</b>.

##### (a)

Given {3,5,8,3,7,2}

The geometric mean $G$ of a set of $N$ numbers $ X_1,X_2,X_3...X_N $ is defined as:

$$
G = \sqrt[N]{X_1X_2X_3...X_N}
$$


Using the given set, we'll have:

$$
G_a = \sqrt[6]{3 \times 5\times 8\times 3\times 7\times 2}
$$

In [43]:
radicand_a = 3*5*8*3*7*2
radicand_a

5040

In [44]:
radicand_a**(1/6)

4.140680833465288

$$
\begin{align*}
G_a &= \sqrt[6]{3 \times 5\times 8\times 3\times 7\times 2} \\
    &= \sqrt[6]{5040}\\
    &\approx 4.14
\end{align*}
$$

##### (b)

Given {28.5, 73.6, 47.2, 31.5, 64.8}

$$
G_b  = \sqrt[5]{28.5\times 73.6 \times 47.2 \times 31.5 \times 64.8}
$$

In [76]:
radicand_b = 28.5*73.6*47.2*31.5*64.8
radicand_b

202092516.864

In [78]:
radicand_b**(1/5)

45.82579906814073

$$
\begin{align*}
G_a &= \sqrt[6]{28.5\times 73.6 \times 47.2 \times 31.5 \times 64.8} \\
    &= \sqrt[6]{202092516.864}\\
    &\approx 45.83
\end{align*}
$$